src/pages/GetInvolved/GetInvolved.js

```diff
@@ -1,10 +1,45 @@
-import React from 'react';
-import Layout from 'components/Layout/Layout';
+import React from "react";
+import { useSelector } from "react-redux";
+import Layout from "components/Layout/Layout";
+import { Link } from "react-router-dom";
+import "./GetInvolved.scss";
 
 const GetInvolved = () => {
+  let clubDataAll = useSelector((state) => state.clubs);
+  let pageInfo = useSelector((state) => state.pages["join-a-club"]);
+
+  if (!pageInfo) {
+    return null;
+  } else {
+    pageInfo = pageInfo.join_a_club_page;
+    pageInfo.body = pageInfo.body.replace(/(<([^>]+)>)/gi, ""); // API object contains html tags here (only). This removes them.
```

Comment:
Why are you removing html from the backend?

Reply:
My fault. Meant to make the paragraph more succinct like in the designs. Changed it to accept html to accept the format.